tools/stimulus/types-RC.yaml

```diff
@@ -76,6 +76,21 @@ CSTRING:
         OUT: |-
             PROTECT(%I% = Rf_mkCharLenCE(%C%, strlen(%C%), CE_UTF8));
 
+DOUBLE:
```

Comment:
What is the purpose of this type and where it is used in `functions.yaml`? It should always be the same as `REAL`.

Reply:
Nope :(